docs/Concepts/Privacy/PrivacyEnhancements.md

```diff
@@ -5,101 +5,170 @@ description: Privacy Enhancements feature prevents nodes from modifying contract
-In order to transact with a contract a node must prove it has access (has the encrypted payload and the ability to decrypt it) to the contract's originating transaction.
-Two flavours of privacy enhanced contracts have been introduced: a) Counter party protection and b) Private state validation.
+* Counter party protection
+* Private state validation
 
-## Counter Party Protection (PP)
+## Counter party protection
 
-This flavor prevents non-party interaction on a private contract but allows state divergence. That is, it will allow nodes to maintain different state through private transaction to "self" or "subset of nodes".
+Counter party protection prevents non-participants sending a transaction to a
+private contract. For example, a private contract is deployed between node 1 and 2. Without counter
+party protection, if node 3 discovers the private contract address, node 3 can send a transaction with
+`privateFor` set to node 2. The transaction changes the private state of node 1 but not node 2 and
+the private states of 1 and 2 diverge.
 
-## Private State Validation (PSV)
+Use the counter party protection instead of access controls on private contract implementations to protect
+against other network participants from updating the state. Counter party protection prevents non-participants
+from interacting with the private contract without additional access controls.
 
-This flavor uses all the verifications of counter party protection. On top of those it introduces further checks preventing nodes from state deviation by failing private transactions to "self" or "subset of nodes" by sharing the full list of recipients among all nodes which is validated against all subsequent transactions (in standard private/party protection mode only the sending node knows the list of recipients).
+## Private state validation
+
+Private state validation prevents state divergence when a private transaction is sent to a subset of
+the transaction participants. For example, a private contract is deployed between node 1 and 2. Without
+private state validation, node 1 can send a transaction to the private contract with a `privateFor`
+of `[]`. The transaction changes the private state of node 1 but not node 2 and the private states of
+1 and 2 no longer match.
+
+When using private state validation, the full participant list is shared among all participants and
+validated against all subsequent transactions. Transactions sent to a subset of participants fail.
+
+In standard privacy or when only using counter party protection, only the sending node knows
+the full participant list.
+
+## Affected Contract Original Transaction Payload Hash
+
+Counter party protection and private state validation are implemented using the Affected Contract
+Original Transaction Payload Hash (`ACOTH`). To transact with a private contract, a node must prove
```

Comment:
maybe define ACOTH at the end of the file as an acronym after PP and PSV.

Reply:
Done